2018-10-24 11:47:50,084 :  INFO [ConsoleApp] :  Le compte 32822621 a réalisé une opération créditer
2018-10-24 11:48:00,606 :  INFO [ConsoleApp] :  Le compte 32822621 a réalisé une opération créditer
2018-10-24 11:48:27,611 :  INFO [ConsoleApp] :  Le compte 31466225 a réalisé une opération créditer
2018-10-24 11:48:27,612 :  INFO [ConsoleApp] :  Le compte 32822621 a réalisé un virement vers 31466225
2018-11-07 19:14:43,746 : ERROR [DBConnection] :  Error while loading database driver
2018-11-07 19:14:46,049 : ERROR [DBConnection] :  Error while loading database driver
2018-11-07 19:14:54,550 : ERROR [DBConnection] :  Error while loading database driver
2018-11-07 19:15:24,573 : ERROR [DBConnection] :  Error while loading database driver
2018-11-07 19:58:44,305 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=COllege 6, cni=A12121, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : java.sql.SQLException: No value specified for parameter 5
2018-11-07 20:00:58,025 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=college med 6, cni=A12346, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : java.sql.SQLException: No value specified for parameter 5
2018-11-07 20:01:37,637 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=college ibn sina imzouren, cni=A123456, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=Boudaa, prenom=Tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: INSERT command denied to user 'tpensah'@'localhost' for table 'compte'
2018-11-07 20:03:24,761 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=college ibn sina, cni=A123456, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: INSERT command denied to user 'tpensah'@'localhost' for table 'compte'
2018-11-07 20:19:23,562 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=College 2, cni=A123456, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=Boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: INSERT command denied to user 'tpensah'@'localhost' for table 'compte'
2018-11-07 20:23:54,854 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=sdsd, cni=A123456, dateNaissance=Wed Dec 12 00:00:00 WET 1235, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: INSERT command denied to user 'tpensah'@'localhost' for table 'compte'
2018-11-07 20:26:01,130 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=col, cni=A123456, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: INSERT command denied to user 'tpensah'@'localhost' for table 'compte'
2018-11-07 20:47:04,077 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=a, cni=a, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: INSERT command denied to user 'tpensah'@'localhost' for table 'compte'
2018-11-07 20:47:41,434 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=a, cni=A, dateNaissance=Wed Dec 12 00:00:00 WET 2018, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Table 'app.compte' doesn't exist
2018-11-07 20:48:59,597 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=0, titulaire=Personne [adresse=a, cni=A, dateNaissance=Thu Dec 12 00:00:00 WET 1258, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : java.sql.SQLException: Field 'num' doesn't have a default value
2018-11-07 20:51:17,798 : ERROR [CompteDAOImpl] :  Error while trying to save the object  Compte [num=34653094, titulaire=Personne [adresse=A, cni=A1254, dateNaissance=Sat Dec 12 00:00:00 WET 1254, nom=boudaa, prenom=tarik], solde=0.0, decouvertMax=800.0, debitMax=3000.0]  due to exception : java.sql.SQLException: Field 'num' doesn't have a default value
2018-11-07 21:19:27,988 : ERROR [CompteDAOImpl] :  Error while trying to save the object  com.bo.Compte@5be6e01c  due to exception : java.sql.SQLException: Field 'num' doesn't have a default value
2018-11-07 21:21:31,382 : ERROR [CompteDAOImpl] :  Error while trying  to find the object with id =   1  due to exception : com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown column 'NUM' in 'where clause'
2018-11-07 21:32:04,494 : ERROR [CompteManager] :  Le compte 1 essaie de faire une Opération impossible car le débit maximal est atteint
2018-11-07 21:35:27,954 : ERROR [CompteManager] :  Le compte 1 essaie de faire une Opération impossible car le débit maximal est atteint
2018-11-16 17:12:48,703 : ERROR [DBConnection] :  Error while trying to connect to database
2018-11-27 09:35:38,716 : ERROR [JDBCConnection] :  Error while trying to connect to database
2018-11-27 09:36:11,696 : ERROR [JDBCConnection] :  Error while trying to connect to database
2018-11-27 09:36:20,780 : ERROR [JDBCConnection] :  Error while trying to connect to database
2018-11-27 09:36:47,601 : ERROR [JDBCConnection] :  Error while trying to connect to database
2018-11-27 09:37:34,608 : ERROR [JDBCConnection] :  Error while trying to connect to database
2018-11-27 09:38:21,255 : ERROR [JDBCConnection] :  Error while trying to connect to database
2018-11-27 09:38:59,463 : ERROR [JDBCConnection] :  Error while trying to connect to database
2018-11-27 09:39:11,403 : ERROR [JDBCConnection] :  Error while trying to connect to database
